kong/plugins/ai-prompt-template/handler.lua

```diff
@@ -129,4 +108,4 @@ function _M:access(conf)
 end
 
 
-return _M
+return plugin
```

Comment:
we tend to go with the plugin's name in camel case + "Handler" here usually, like:
```suggestion
return AIPromptTemplateHandler
```